source/gent/tier1_hortense.rst

```diff
@@ -617,3 +634,37 @@ Getting help
 
 For questions and problems related to Tier-1 Hortense, please contact the central
 support address for Tier-1 compute: `compute@vscentrum.be <mailto:compute@vscentrum.be>`_.
+
+
+Phase 2
```

Comment:
```suggestion
Phase 2 (May 2023)
```

Maybe this should be under a separate "History" section, where you also mention something on `Phase 1`?